fe1-web/store/reducers/SocialReducer.ts

```diff
@@ -64,6 +126,8 @@ export const {
   addChirp,
 } = socialSlice.actions;
 
+export const socialReducer = socialSlice.reducer;
```

Comment:
Doesn't it create issues ? Because socialReducer is already defined in RootReducer.
```suggestion
export const socialReduce = socialSlice.reducer;
```

Reply:
Indeed it creates an issue but I need it for testing, this is also the reason I added "import/no-named-as-default": 0 in eslintrc. I don't understand why the wallet reducer is doing the same thing but works without any problem.